src/language-markdown/utils.js

```diff
@@ -162,6 +162,34 @@ function getOrderedListItemInfo(orderListItem, originalText) {
   return { numberText, marker, leadingSpaces };
 }
 
+function hasGitDiffFriendlyOrderedList(node, options) {
```

Comment:
why not name `hasDiffFriendlyOrderedList`

Reply:
Because it is git diff, not diff, it is original name, so it is out of this PR